conda-recipe/run_test.sh

```diff
@@ -18,7 +18,18 @@
 # limitations under the License.
 #*******************************************************************************
 
-python -c "import daal4py"
-mpirun -n 4 python -m unittest discover -v -s tests -p spmd*.py
-pytest --pyargs daal4py/sklearn/
-python tests/run_tests.py
+ok=0
+
+if ! python -c "import daal4py"; then
+    ok=1
+fi
+if ! mpirun -n 4 python -m unittest discover -v -s tests -p spmd*.py; then
```

Comment:
daal CI have build without MPI support. So, we need to skip MPI check in this case. For example:
```python
if ! $NO_DISTR; then
...
fi
```